pkg/api/v1beta3/dynakube/logmonitoring_props.go

```diff
@@ -0,0 +1,18 @@
+package dynakube
+
+import (
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube/logmonitoring"
+)
+
+func (dk *DynaKube) LogMonitoring() *logmonitoring.LogMonitoring {
+	lm := &logmonitoring.LogMonitoring{
+		Spec: dk.Spec.LogMonitoring,
+	}
+	lm.SetName(dk.Name)
+
+	return lm
+}
+
+func (dk *DynaKube) LogMonitoringTemplates() logmonitoring.TemplateSpec {
+	return dk.Spec.Templates.LogMonitoring
+}
```

Comment:
do it more like 

in `pkg/api/v1beta3/dynakube/logmonitoring/spec.go`
```
type LogMonitoring struct {
	*Spec
	*Templates

	name string
}
```

in `pkg/api/v1beta3/dynakube/logmonitoring_props.go`

```
func (dk *DynaKube) LogMonitoring() *logmonitoring.LogMonitoring {
	lm := &logmonitoring.LogMonitoring{
		Spec: dk.Spec.LogMonitoring,
		Templates: dk.Spec.Templates.LogMonitoring
	}
	lm.SetName(dk.Name)

	return lm
}
```

So you can do the following `dk.LogMonitoring().DNSPolicy` and such.
IMO, it is much nicer than `dk. LogMonitoringTemplates().DNSPolicy`
Or you can also do `dk.LogMonitoring().Templates.DNSPolicy` if there is naming conflict

Reply:
if there ever be a `logmonitoring.Status` you would also put that into the `type LogMonitoring struct`, so we truly handle it as their own CR

Reply:
👍  - Done